provision/test-data.json

```diff
@@ -99,6 +99,6 @@
 {"index": {"_type": "meta", "_id": "c06232ff-f4cd-42d2-ba6e-2cdad82cb7cb.2018-06-22T170021.367173Z", "_index": "fb_index"}}
 {"repoDataBundleId": "c06232ff-f4cd-42d2-ba6e-2cdad82cb7cb", "center_name": "Broad", "submitter_donor_id": "", "sampleId": "e7741c77-6d8d-46bd-af8f-3b2434e3ca3a", "submittedSampleId": "NWD231092", "repoBaseUrl": "", "analysis_type": "", "repoCode": "DSS-AWS-Oregon", "aliases": ["dg.4503/c06232ff-f4cd-42d2-ba6e-2cdad82cb7cb"], "repoOrg": "UCSC", "file_type": "crai", "fileMd5sum": "8368645be5bc9d2993fd49e15a3432fe8b8265eb", "file_version": "2018-06-22T170021.367173Z", "access": "public", "submittedSpecimenId": "NWD231092", "program": "TOPMed", "repoType": "HCA DSS", "repoName": "DSS-AWS-Oregon", "donor": "6d13d82a-aa6c-4a3c-8f3d-528899391043", "workflowVersion": "", "experimentalStrategy": "", "download_id": "c06232ff-f4cd-42d2-ba6e-2cdad82cb7cb", "repoCountry": "US", "workflow": "", "fileSize": 5618, "submittedDonorId": "HG01111", "specimen_type": "Normal - blood", "metadataJson": "", "lastModified": "2018-06-22T170021.367173Z", "study": "topmed-public", "submitterDonorPrimarySite": "B-lymphocyte", "specimenUUID": "f2555751-d329-4b81-a0e9-7989562c860d", "project": "topmed-public", "file_id": "c06232ff-f4cd-42d2-ba6e-2cdad82cb7cb", "redwoodDonorUUID": "", "urls": ["gs://topmed_workflow_testing/topmed_aligner/input_files/NWD231092.0005.recab.cram.crai"], "title": "NWD231092.0005.recab.cram.crai", "software": ""}
 {"index": {"_type": "databundle", "_id": "100", "_index": "db_index"}}
-{"id": "1", "version": "2", "checksums": ["md5:8368645be5bc9d2993fd49e15a3432fe8b8265eb"], "updated": "today", "created": "today", "description": "test", "aliases": ["test_alias_1:test_alias_value"]}
+{"id": "1", "version": "2", "checksums": ["md5:8368645be5bc9d2993fd49e15a3432fe8b8265eb"], "data_object_ids": ["b7c6ad18-070a-4721-a464-e61a1137f539", "b7f70d3b-b660-4537-97ba-0b352ea4a5dd"], "updated": "today", "created": "today", "description": "test", "aliases": ["test_alias_1:test_alias_value"]}
```

Comment:
Thanks for doing this, I know it seems arbitrary, but it makes the demonstration make more sense!